### ISSUE TITLE
Add different folders to put download

### ISSUE THREAD

**Issue by TuXbOy06** [pyload/pyload#3606]
Hi,
I think can that it be a useful to be able to choose in which repository on the server or PC we want to put our downloads...

For example:

To be able to put music to /opt/pyload/downloads/music
To be able to put movie to /opt/pyload/dowloads/movies
To be able to put example to /opt/pyload/dowloads/example

Want do you think of my idea?

**Comment by OptimusGREEN** [comment]
Yes I'm all for this. It would be good just to have an optional dl_dest_path that is passed as an arg to addpackage and reverts to default downloads folder if not available or not passed.

**Comment by mihawk90** [comment]
I know this issue is old but it popped up during a search for my own issue. This might have been added in the meantime but you can already do this.

When you edit the package (via :pencil2: ) you can enter a path with a `/` and it will create subfolders. So suppose your package is named `Package`, you can just enter `music/package` and it will create (if it not exists) a folder `music` and a sub-folder `package` where your files go.

It's not suuuuper convenient, but it's possible.
